pkg/standalone/list.go

```diff
@@ -134,6 +135,11 @@ func List() ([]ListOutput, error) {
 				continue
 			}
 
+			httpReadBufferSize, err := strconv.Atoi(argumentsMap["--dapr-http-read-buffer-size"])
+			if err != nil {
+				continue
```

Comment:
@shubham1172 I have a quick question here ... Since you are also working on one Pr regarding listing of daprd instances ... 
If any of the parameters gives an error, that whole process is skipped from the list, is that the behavior that we want ?

Reply:
@mukundansundar this behavior is changed in my PR. Example:

**Original behavior**
https://github.com/dapr/cli/blob/0c748aba8607e67c5339bd92855647c8a9aa7284/pkg/standalone/list.go#L102-L105

**New behavior**
https://github.com/dapr/cli/blob/acc9702164211a0e6dcd5313bcf108adfc2a1e87/pkg/standalone/list.go#L88-L93

Once this PR is merged with master, I will update the behavior in my PR #959 too.